cmd/cli/main.go

```diff
@@ -0,0 +1,21 @@
+package main
```

Comment:
I separated the `chatto` server from the `cli`. So now we can run the cli locally and connect to the server remotely.

Reply:
Very good idea